server/routes.js

```diff
@@ -112,12 +112,29 @@ module.exports = [{
     }
   }
 }, {
-  // static routes using dist/, yay grunt
   method: 'GET',
-  path: '/assets/{param*}',
+  path: '/images/{param*}',
   handler: {
     directory: {
-      path: path.join(__dirname, '..', config.get('server.staticPath')),
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'images'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/scripts/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'scripts'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/styles/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'styles'),
```

Comment:
This represents an opposing view, but I was just thinking we'd change it to this:

`path: '/{param*}'`

I think we should feel comfortable serving anything from public since it's name kind of implies that it is in fact public. If we do want something more specific, it seems like we could construct one fancy route to cover all the cases.